### ISSUE TITLE
Forbidden Motion

### ISSUE THREAD

**Issue by roefer** [RoboCup-SPL/Rules#141]
The forbidden motion penalty is unbalanced. Teams get a 30 seconds "penalty" for ignoring the referee signal, but only a 15 seconds penalty for detecting one too early. Since the penalty time will be counting down from the start of the Ready state, it is an advantageous approach to intentionally getting the penalty, because when unpenalized, 30 s are left to walk to the kick-off positions, which is better than risking to only have 15 seconds left.

**Comment by ben47955** [comment]
> Robots that begin moving their legs or locomote in any fashion before the specified state in the
subsections will be penalized in place on the field for 15 s in playing.

This text cover both motion in initial and set from rulebook. This mean a motion in Initial result in being penalized the whole ready phase and 15s in playing on the side. This is 30 seconds later than doing nothing. Maybe this isn't clear and I will see if we can add a example.

**Comment by roefer** [comment]
You are right, the rules state that the penalty is applied in Playing. However, they do not say what happens between Initial and Playing.

**Comment by ben47955** [comment]
We will add clarification. Thanks for reporting.
